Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -92,7 +92,9 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
     let initialDataUnpacked = Signal.zip(project, baseReward, refTag, context)
 
     let backing = project.map { $0.personalization.backing }.skipNil()
-    self.showPledgeOverTimeUI = project.signal.map { $0.isPledgeOverTimeAllowed ?? false }
+    self.showPledgeOverTimeUI = project.signal
+      .map { ($0.isPledgeOverTimeAllowed ?? false) && featurePledgeOverTimeEnabled()
+      }
```

Comment:
nit: format here is weird to me with the closing bracket on its own line. Can you run `bin/format.sh` to make sure it doesn't want to change it?

I'd either do `.map { ($0.isPledgeOverTimeAllowed ?? false) && featurePledgeOverTimeEnabled() }` all on one line or 
```
.map {
  ($0.isPledgeOverTimeAllowed ?? false) && featurePledgeOverTimeEnabled()
}
```
spread out over 3 lines.